src/p8est_search.h

```diff
@@ -454,6 +454,43 @@ void                p8est_search_partition_gfx
    p8est_search_partition_t quadrant_fn, p8est_search_partition_t point_fn,
    sc_array_t *points);
 
+/** Traverse some given global partition top-down.
+ * The partition can be that of any p4est, not necessarily known to the
+ * caller.  This is not a collective function.  It does not communicate.
+ * We proceed top-down through the partition, identically on all processors
+ * except for the results of two user-provided callbacks.  The recursion will only
+ * go down branches that are split between multiple processors.  The callback
+ * functions can be used to stop a branch recursion even for split branches.
+ * This function offers the option to search for arbitrary user-defined points
+ * analogously to \ref p4est_search_local.
+ * This function is similar to p4est_search_partition_gfx, but does not require
```

Comment:
You may use the \ref doxygen command to generously cross-link functions. I would use it wherever a linkable prototype occurs. Please check the generated html for consistent looks and remove warnings as you go.

Reply:
Good point, I added several references to the documentation.